processor/tailsamplingprocessor/processor.go

```diff
@@ -77,14 +77,15 @@ type spanAndScope struct {
 }
 
 var (
-	attrSampledTrue     = metric.WithAttributes(attribute.String("sampled", "true"))
-	attrSampledFalse    = metric.WithAttributes(attribute.String("sampled", "false"))
-	decisionToAttribute = map[sampling.Decision]metric.MeasurementOption{
+	attrSampledTrue      = metric.WithAttributes(attribute.String("sampled", "true"), attribute.String("dropped", "false"))
+	attrSampledFalse     = metric.WithAttributes(attribute.String("sampled", "false"), attribute.String("dropped", "false"))
+	attrDropped          = metric.WithAttributes(attribute.String("sampled", "false"), attribute.String("dropped", "true"))
```

Comment:
Attributes on emitted metrics should be included in the metadata.yaml like https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1f92d42bea48e3fcbb351f95340563990001bd2a/receiver/kubeletstatsreceiver/metadata.yaml#L73-L81. Can you update it with the `sampled` and `dropped` attributes?

Reply:
Ah, I wasn't aware this exists, thanks!